3460/0/about_scoring_project.rb

```diff
@@ -0,0 +1,87 @@
+# frozen_string_literal: true
+
+require File.expand_path(File.dirname(__FILE__) + '/neo')
+
+# Greed is a dice game where you roll up to five dice to accumulate
+# points.  The following "score" function will be used to calculate the
+# score of a single roll of the dice.
+#
+# A greed roll is scored as follows:
+#
+# * A set of three ones is 1000 points
+#
+# * A set of three numbers (other than ones) is worth 100 times the
+#   number. (e.g. three fives is 500 points).
+#
+# * A one (that is not part of a set of three) is worth 100 points.
+#
+# * A five (that is not part of a set of three) is worth 50 points.
+#
+# * Everything else is worth 0 points.
+#
+#
+# Examples:
+#
+# score([1,1,1,5,1]) => 1150 points
+# score([2,3,4,6,2]) => 0 points
+# score([3,4,5,3,3]) => 350 points
+# score([1,5,1,2,4]) => 250 points
+#
+# More scoring examples are given in the tests below:
+#
+# Your goal is to write the score method.
+# rubocop:disable Lint/UselessAssignment
```

Comment:
not needed here

Reply:
the dog begins to swear
![Screenshot from 2019-07-08 23-50-05](https://user-images.githubusercontent.com/49589929/60841919-9880d300-a1db-11e9-98d8-d9194e6e2d76.png)